bin/make_err

```diff
@@ -213,18 +213,98 @@ sub create_public ($) {
     close HEADER;
 }
 
+##############################################################################
+# Create the generated portion of the H5E major message definition code
+#
+sub create_majdef ($) {
```

Comment:
This creates a static const message for each library-internal major error message, so they aren't malloc'ed/free'd at init/term.